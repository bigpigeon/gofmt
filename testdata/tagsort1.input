//gofmt -ts
package main
type Example struct {
	Data string `xml:"data" yaml:"data"  json:"data"  `
}