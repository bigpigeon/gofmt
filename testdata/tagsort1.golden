//gofmt -ts
package main

type Example struct {
	Data string `json:"data" xml:"data" yaml:"data"`
}
